meta-date.sh

```diff
@@ -0,0 +1,2 @@
+CURRENT_DATE=$(date +‘%d/%m/%Y’)
```

Comment:
open graph datetime suggests iso 8601 so double digits for all dates https://ogp.me/#datetime
(We could also include the time)

Can validate with https://developers.facebook.com/tools/debug/
```
date +"%Y-%m-%dT%H:%M:%S%z"
2020-03-21T20:34:05+1100
```